wallet/package.json

```diff
@@ -86,7 +86,7 @@
     "analyze": "source-map-explorer 'build/static/js/*.js'",
     "start": "yarn build:ses && react-app-rewired start",
     "build": "yarn build:ses && yarn build:react",
-    "build:ses": "cp node_modules/ses/dist/lockdown.umd.js public/",
+    "build:ses": "cp ../node_modules/ses/dist/lockdown.umd.js public/",
```

Comment:
_question_

Just curious, is this script still needed?

Reply:
Yea, the wallet-app is structured differently from the other apps, and does lockdown in index.html https://github.com/Agoric/wallet-app/blob/main/wallet/public/index.html#L5-L21

IIRC it was done so that the bundler would not transform `lockdown.umd.js`, which was causing issues at the time. The newer apps that use vite don't have this problem.

Reply:
I see, thanks for the explanation